### PR TITLE
SKALE-4839 Handle EC=0 in RPC check, wip

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: tests/schains/monitor/rpc_test.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        entrypoint='bash -c "exit 0"'
+    )
+    sleep(10)
+
+    assert not monitor_schain_rpc(
+        schain={'name': schain_db},
+        schain_record=schain_record,
+        dutils=dutils
+    )
+    assert 'container exited with zero, skipping RPC monitor' in caplog.text
```
Comment (badrogger) [inline tests/schains/monitor/rpc_test.py]: I think we should add container status check here. It's more reliable than checking logs output.

File: tests/schains/monitor/rpc_test.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+def test_monitor_schain_rpc_ec_0(caplog, schain_db, dutils):
+    schain_record = SChainRecord.get_by_name(schain_db)
+
+    image_name, container_name, _, _ = get_container_info(
+        SCHAIN_CONTAINER, schain_db)
+
+    dutils.run_container(
+        image_name=image_name,
```
Comment (badrogger) [inline tests/schains/monitor/rpc_test.py:29]: The container isn't removed anywhere.

**Review by badrogger — CHANGES_REQUESTED** [review]

File: tests/schains/monitor/rpc_test.py (hunk 1 above)
Comment (dmytrotkk) [reply tests/schains/monitor/rpc_test.py]: updated tests

File: tests/schains/monitor/rpc_test.py (hunk 2 above)
Comment (dmytrotkk) [reply tests/schains/monitor/rpc_test.py:29]: addressed

**Review by badrogger — APPROVED** [review]

**Pull request merged**